nixos/common/default.nix

```diff
@@ -16,6 +16,13 @@
     ./zfs.nix
   ];
 
+  system.switch = {
+    enable = lib.mkDefault false;
```

Comment:
@Mic92 what happens when this is set to false and enableNg is unset?

Reply:
Then you can't switch. That's what the perlless tests are using.

Reply:
Ah hang on. This is actually pretty bad if you are on an old version.

Reply:
Fixed in https://github.com/nix-community/srvos/pull/433/files